### ISSUE TITLE
Event page

### ISSUE THREAD

**Issue by schalkneethling** [mdn/developer-portal#1723]
A couple of mostly structural issues to address on this page:

1. [x] The featured section's heading hierarchy is incorrect. The structure should be something like:

```
<div class="mzp-l-content">
  <h2>Featured</h2>

  <ul>
    <li class="mzp-c-card mzp-c-card-extra-small mzp-has-aspect-16-9">
      <a href="/events/oxidize-global-2020/" class="mzp-c-card-block-link" data-type="event">
        ...
        <h3 class="mzp-c-card-title">Oxidize Global 2020</h3>
      </a>
    <li>
  </ul>
</div>
```

2. [x] The community events section's heading hierarchy is incorrect. The structure should be something like:

```
<div class="mzp-l-content events-middle-content">      
  <div class="rich-text">
    <h2>Community Events</h2>
```

Each entry in the results list should then use an `h3` element as its heading.

3. [ ] The results list should be ideally marked up as a list

**Comment by valgrimm** [comment]
Val has fixed "The community events section's heading hierarchy is incorrect"

**Comment by stevejalim** [comment]
> The featured section's heading hierarchy is incorrect. The structure should be something like:

This was addressed in as part of #1736

**Comment by stevejalim** [comment]
Only thing remaining is:

> The results list should be ideally marked up as a list

Which is the filter-list.html used on Posts, Events and People

**Comment by stevejalim** [comment]
https://github.com/mdn/developer-portal/pull/1761 addresses `The results list should be ideally marked up as a list`
